templates/python/package/aio/client.py.twig

```diff
@@ -0,0 +1,191 @@
+import io
+import httpx
+import os
+from .input_file import InputFile
+from .exception import {{spec.title | caseUcfirst}}Exception
+
+class Client:
```

Comment:
Could we use the existing client as a base class and extend here as a child `AsyncClient`? Then we would only need to maintain the async methods in the `aio` package

Reply:
Updated to create a AsyncClient (inherits Client). Replaces ``call`` and ``chunked_upload`` methods.